base-images/python/Dockerfile

```diff
@@ -30,6 +30,8 @@ ENV PYTHONUNBUFFERED 1
 # source /env/bin/activate. This ensures the application is executed within
 # the context of the virtualenv and will have access to its dependencies.
 ENV VIRTUAL_ENV /env
+# Python grpc libraries should use the host's native DNS resolution
```

Comment:
Just a side note - I would like to call this out whenever we write this up for a blog post, because it was such a monumental pain to me when I was trying to test. Given the issues with DNS and microservices, maybe this warrants its own post as part of the series on MARA (not just this aspect, but others as well)

Reply:
Yeah, I had to spend some time reading through grpc code to find that.